qiskit/circuit/library/data_preparation/_zz_feature_map.py

```diff
@@ -19,7 +19,7 @@
 
 
 class ZZFeatureMap(PauliFeatureMap):
-    r"""Second-order Pauli-Z evolution circuit.
+    r"""Second-order Pauli-Z evolution circuit: Using the Pauli-Z gate to create entanglement between pairs of qubits allowing them to evolve.
```

Comment:
The first line is just a very brief summary, I would keep it as is 🙂

```suggestion
    r"""Second-order Pauli-Z evolution circuit.
```

Reply:
Okay. I will revert it back. I modified it because it was unclear to me what it meant, so I thought it could be unclear to others as well.